website/content/en/status/report-2024-04-2024-06/ci.adoc

```diff
@@ -0,0 +1,51 @@
+=== Continuous Integration
+
+Links: +
+link:https://ci.FreeBSD.org[FreeBSD Jenkins Instance] URL: link:https://ci.FreeBSD.org[] +
+link:https://tinderbox.freebsd.org[FreeBSD CI Tinderbox view] URL: link:https://https://tinderbox.freebsd.org[] +
+link:https://artifact.ci.FreeBSD.org[FreeBSD CI artifact archive] URL: link:https://artifact.ci.FreeBSD.org[] +
+link:https://wiki.FreeBSD.org/HostedCI[Hosted CI wiki] URL: link:https://wiki.FreeBSD.org/HostedCI[] +
+link:https://wiki.FreeBSD.org/3rdPartySoftwareCI[3rd Party Software CI] URL: link:https://wiki.FreeBSD.org/3rdPartySoftwareCI[] +
+link:https://bugs.freebsd.org/bugzilla/buglist.cgi?bug_status=__open__&email1=testing%40FreeBSD.org&emailassigned_to1=1&emailcc1=1&emailtype1=equals[Tickets related to freebsd-testing@] URL: link:https://bugs.freebsd.org/bugzilla/buglist.cgi?bug_status=__open__&email1=testing%40FreeBSD.org&emailassigned_to1=1&emailcc1=1&emailtype1=equals[] +
+link:https://github.com/freebsd/freebsd-ci[FreeBSD CI Repository] URL: link:https://github.com/freebsd/freebsd-ci[] +
+link:https://lists.FreeBSD.org/subscription/dev-ci[dev-ci Mailing List] URL: link:https://lists.FreeBSD.org/subscription/dev-ci[]
+
+Contact: Jenkins Admin <jenkins-admin@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+Contact: link:https://lists.FreeBSD.org/mailman/listinfo/freebsd-testing[freebsd-testing Mailing List] +
+Contact: IRC #freebsd-ci channel on EFNet
+
+In the second quarter of 2024, we worked with the project contributors and developers to address their testing requirements.
+Concurrently, we collaborated with external projects and companies to enhance their products by testing more on FreeBSD.
+
+Important completed tasks:
+* Added new hardware purchased by the FreeBSD Foundation in Chicago colo to the CI cluster
+* Repurposed decommissioned pkg builder as build agent and added to the CI cluster
+* Adjusted the job dispatching mechanism based on the machine capbilitiy
+* bofh@ merged link:https://reviews.freebsd.org/D43786[] as https://cgit.freebsd.org/src/commit/?id=cb9d4bb1fbb9ac0eb9f211656e91f9d5254c166c[Add preliminary in-tree CI infrastructure for developers] so they can replicate an environment or
+results similar to https://ci.FreeBSD.org[]
+* Decommission armv6 jobs for the main branch
+
+Work in progress tasks:
+
+* Merging link:https://reviews.freebsd.org/D36257[]
```

Comment:
The review is not accessible to who does not have a Phabricator account, so let's specify its title.

```suggestion
* Merging link:https://reviews.freebsd.org/D36257[Pre-commit CI with CIRRUS-CI]
```